libs/sdk-ui-dashboard/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+    "_note": "This file is only for IDE. For build use tsconfig.[dev/build].json (dev mode is less strict)",
+    "compilerOptions": {
+        "strict": true,
+        "declaration": true,
+        "declarationMap": true,
+        "target": "es5",
+        "jsx": "react",
+        "lib": ["es7", "dom"],
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "esModuleInterop": true,
+        "importHelpers": true,
+        "noImplicitAny": true,
```

Comment:
Please add "strict": true if possible

Reply:
its there.. right on the top :) but will move it down as that's where we usually put it